go/vt/vtctl/grpcvtctldserver/server.go

```diff
@@ -2438,6 +2438,55 @@ func (s *VtctldServer) GetUnresolvedTransactions(ctx context.Context, req *vtctl
 	}, nil
 }
 
+// ConcludeTransaction is part of the vtctlservicepb.VtctldServer interface.
+// It concludes the unresolved distributed transaction.
+func (s *VtctldServer) ConcludeTransaction(ctx context.Context, req *vtctldatapb.ConcludeTransactionRequest) (resp *vtctldatapb.ConcludeTransactionResponse, err error) {
+	span, ctx := trace.NewSpan(ctx, "VtctldServer.ConcludeTransaction")
+	defer span.Finish()
```

Comment:
Do we want to add the dtid and participants to the trace span?

Reply:
I think we can add that